src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs

```diff
@@ -120,7 +120,7 @@ public virtual void SetupLayer()
 
 			_actualView.Layer.RasterizationScale = UIScreen.MainScreen.Scale;
 			_actualView.Layer.ShouldRasterize = true;
-			_actualView.Layer.MasksToBounds = Element.IsClippedToBounds;
+			_actualView.Layer.MasksToBounds = Element.IsSet(Controls.Compatibility.Layout.IsClippedToBoundsProperty) ? Element.IsClippedToBounds : true;
```

Comment:
It looks like Android has almost the same logic

![image](https://user-images.githubusercontent.com/5375137/202508347-b80f5b3c-b7f7-4a37-b639-cf15a3e934d1.png)

can we consolidate these two calls into an extension method? 

Maybe something like this? 

`bool IsClippedBounds(this Layout layout)`

Also, I didn't see this section on Windows which is curious :-) 
I don't think we should hold up this PR on Windows but I'm wondering why this code isn't on the windows `FrameREnderer`

Reply:
Thanks for pointing that out! I didn't notice that! 
I updated the PR to use an extension method for android and ios and will take a further look into windows